benchmark/clblast/CMakeLists.txt

```diff
@@ -29,10 +34,9 @@ foreach(clblast_benchmark ${CLBLAST_BENCH_SRCS})
   add_executable(bench_clblast_${clblast_bench_exec} ${clblast_benchmark} ${CLBLAST_BENCH}/main.cpp ${CLBLAST_BENCH}/clwrap.cpp)
   set_target_properties(bench_clblast_${clblast_bench_exec} PROPERTIES CXX_STANDARD 11)
   add_dependencies(bench_clblast_${clblast_bench_exec} benchmark)
-  target_link_libraries(bench_clblast_${clblast_bench_exec} PUBLIC ${OpenCL_LIBRARIES})
-  target_link_libraries(bench_clblast_${clblast_bench_exec} PUBLIC benchmark)
+  target_link_libraries(bench_clblast_${clblast_bench_exec} PUBLIC benchmark ${OpenCL_LIBRARIES} ${BLAS_LIBRARIES})
   target_link_libraries(bench_clblast_${clblast_bench_exec} PRIVATE clblast)
-  target_include_directories(bench_clblast_${clblast_bench_exec} PUBLIC ${ComputeCpp_INCLUDE_DIRS} ${SYCLBLAS_INCLUDE} ${CLARA_INCLUDE})
+  target_include_directories(bench_clblast_${clblast_bench_exec} PUBLIC ${SYCLBLAS_INCLUDE} ${THIRD_PARTIES_INCLUDE} ${BLAS_INCLUDE_DIRS})
```

Comment:
What is {BLAS_INCLUDE_DIRS}? if it is a third party include it should be added there

Reply:
Because `BLAS_INCLUDE_DIRS` is defined in `FindSystemBLAS.cmake` (or empty if the verification is not activated).
The third party includes are defined in the top level cmake file